public/ktx2.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
I guess because `ktx2.h` is used by both renderer and imagelib, it's being put to `public`? libpublic only contains basic C string, math and buildinfo functions.

imagelib is planned to be split into separate library, so you can store such header in img_ktx2.h in imagelib itself, and include from renderer. Renderers can include private engine headers anyway, except functions, for obvious reason.

Also you missed copyright header here.